src/main/assembly/bin/sonar-scanner.bat

```diff
@@ -65,6 +65,9 @@ goto run
 
 set PROJECT_HOME=%CD%
 
+@REM remove trailing backslash, see https://groups.google.com/d/msg/sonarqube/wi7u-CyV_tc/3u9UKRmABQAJ
+IF %PROJECT_HOME:~-1%==\ SET PROJECT_HOME=%PROJECT_HOME:~0,-1%
```

Comment:
Please add spaces around the `==` operator, to be like this:

    IF %PROJECT_HOME:~-1% == \ SET PROJECT_HOME=%PROJECT_HOME:~0,-1%